crates/types/src/light_client.rs

```diff
@@ -3,9 +3,12 @@
 use ark_ed_on_bn254::EdwardsConfig as Config;
 use ark_ff::PrimeField;
 use ark_serialize::{CanonicalDeserialize, CanonicalSerialize};
+use ethereum_types::U256;
```

Comment:
we should use `ethers_core::types::U256` instead (crate [here](https://crates.io/crates/ethers-core))